lib/msf/core/payload/windows/x64/reverse_http_x64.rb

```diff
@@ -122,7 +122,7 @@ def generate_uri(opts={})
   # Generate the URI for the initial stager
   #
   def generate_small_uri
-    generate_uri_uuid_mode(:init_native, 5)
+    generate_uri_uuid_mode(:init_native, 30)
```

Comment:
See https://github.com/rapid7/metasploit-framework/pull/9321
The fix was added to the x86 payload, here, only six years later, we're adding it to the x64 payload.